### PR TITLE
sourcegraph 0.11.2 (new formula)

### PR DESCRIPTION
**Pull request by uforic**

### CODE REVIEW

File: Library/Formula/sourcegraph.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  homepage "https://www.sourcegraph.com"
+  url "https://sourcegraph.com/.download/0.11.2/darwin-amd64/src.gz"
+  sha256 "452d2c1710efeb3c42160280c14e261c14c2de903f49cacd0d2e4bb426c48b8a"
+
+  def install
+    bin.install "src"
+  end
+
+  test do
+    `system "#{bin}/src" "help"`
```
Comment (UniqMartin) [inline Library/Formula/sourcegraph.rb]: This look wrong. I think you meant this to be `system "#{bin}/src", "help"` (without the backticks). Also, can this test be modified to do something more substantial than just printing the help text? Thanks!

File: Library/Formula/sourcegraph.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class Sourcegraph < Formula
+  desc "Smarter code host powering top-tier development teams"
```
Comment (UniqMartin) [inline Library/Formula/sourcegraph.rb]: This sounds more like marketing-speech than an actual description of what the tool does. Can you improve this please?

File: Library/Formula/sourcegraph.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  homepage "https://www.sourcegraph.com"
+  url "https://sourcegraph.com/.download/0.11.2/darwin-amd64/src.gz"
+  sha256 "452d2c1710efeb3c42160280c14e261c14c2de903f49cacd0d2e4bb426c48b8a"
+
+  def install
+    bin.install "src"
```
Comment (UniqMartin) [inline Library/Formula/sourcegraph.rb]: Is this a monolithic pre-compiled binary? If so, we're probably not going to accept it, see [Acceptable Formulae](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Acceptable-Formulae.md#we-dont-like-binary-formulae).

File: Library/Formula/sourcegraph.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+class Sourcegraph < Formula
+  desc "Smarter code host powering top-tier development teams"
+  homepage "https://www.sourcegraph.com"
+  url "https://sourcegraph.com/.download/0.11.2/darwin-amd64/src.gz"
```
Comment (bfontaine) [inline Library/Formula/sourcegraph.rb]: We need to build this from source.

**Comment by bfontaine** [comment]
According to [its README](https://src.sourcegraph.com/sourcegraph@master/.tree/README.md) it’s under the [Fair Source License](https://fair.io/).
> The Fair Source License is not an open-source license and doesn’t intend to be an open-source license.

We can’t accept software that’s not open-source in the core repo, sorry! You can however maintain it in your own tap: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md
Thanks!

**Pull request closed without merging**